service/storage_service.cc

```diff
@@ -5239,6 +5244,9 @@ future<> storage_service::move_tablet(table_id table, dht::token token, locator:
         auto last_token = tmap.get_last_token(tid);
         auto gid = locator::global_tablet_id{table, tid};
 
+        if (tmap.get_tablet_transition_info(tid)) {
+            throw std::runtime_error(format("Tablet {} is being transmitted", gid));
```

Comment:
Nit about wording: "Tablet {} is in transition". "transmission" is something different and we don't use this concept with tablets.

Reply:
ahh, thank you for this. will keep this in mind. fixed.